### DISCUSSION TITLE
Cura Install

### DISCUSSION THREAD

**Discussion by Grafton1046** [kageurufu/cancelobject-works#6] [Q&A]
I attempted to install the preprocesss script into cura in the same manner as https://github.com/Annex-Engineering/klipper_estimator
by adding the python scritp to the scripts folder and putting the exe in there as well.

I'm not seeing the option to add the preprocess to the post processing steps in though it does seem cura has seen the script and generated a compiled python file.

**Comment by Gil80** [comment]
I'm having the same issue. It's not clear how to install this in Cura.

**Comment by Dnic94** [comment]
Hey,

I created a small script that calls the preprocessor while saving the gcode file.
Please see my PR #15
